### PR TITLE
Make balena-cli build on refreshed on-prem workers

### PR DESCRIPTION
**Pull request by ab77**
** do not ** merge until balenaCI is migrated fully to Concourse v6 PR

    Makes balena-cli build on refreshed on-prem workers
    
    * Fix 'balena ssh' test cases when using the Windows built-in ssh tool
    * Fix Windows installer build in new balena CI workers (qqjs patch)
    * Remove hardcoded path
    
    Change-type: patch

### CODE REVIEW

**Comment by ab77** [comment]
@pdcastro I think I finally worked the rebase out...

**Comment by ab77** [comment]
@balena-ci retest

**Comment by ab77** [comment]
@nazrhom FYI - requires merging before we switch on balena-io pipeline in Concourse v6

File: automation/utils.ts [hunk 1]
```diff
@@ -20,7 +20,8 @@ import * as _ from 'lodash';
 import * as path from 'path';
 import * as shellEscape from 'shell-escape';
 
-export const MSYS2_BASH = 'C:\\msys64\\usr\\bin\\bash.exe';
+export const MSYS2_BASH =
+	process.env.MSYSSHELLPATH || 'C:\\msys64\\usr\\bin\\bash.exe';
```
Comment (pdcastro) [inline automation/utils.ts:24]: For the record, this is not a new env var: it matches the existing var used here:
https://github.com/balena-io/balena-cli/blob/v12.30.3/patches/all/qqjs%2B%2Bexeca%2B0.10.0.dev.patch#L29

**Comment by ab77** [comment]
@balena-ci retest

**Comment by srlowe** [comment]
Pedantic nitpick: The commit message would be better as "Make.." rather than "Makes.." to fit release notes conventions.

**Review by pdcastro — APPROVED** [review]

**Review by pdcastro — APPROVED** [review]

**Pull request merged**